packages/api/app/Models/User.js

```diff
@@ -11,6 +11,7 @@ const Hash = use('Hash');
 const Encryption = use('Encryption');
 
 const { roles } = require('../Utils/roles_capabilities');
+const { lattes: lattesBaseUrl } = require('../Utils/baseUrls');
```

Comment:
See the comment above
```suggestion
const Config = use('Config');
```

Reply:
Done 👍🏻